### ISSUE TITLE
Mark as 1.19.4-compatible

### ISSUE THREAD

**Issue by Madis0** [hamarb123/MCMacOSInputFixes#11]
You said in CF comments that it works on 1.19.4, but it is not marked as such on CurseForge or Modrinth. Please mark 1.19.4 as a supported version.

**Comment by hamarb123** [comment]
Copy of my reply on CF to the person that originally asked the question:

Sorry for late (and long) reply, the version 1.4 should work on 1.19.4, I haven't marked it as such though since 1.19.4 wasn't out when I released version 1.4. If the mod were to not work on any future version, it would likely be the interface options it adds on the mouse options screen that would break / crash, but the core functionality would either just work or be obviously broken (likely crashing also). I also don't limit it to versions I've specifically tested as it annoys me greatly when other mods fail to load, when they would likely work just fine or crash otherwise (which they often do when I go and edit their file), meaning that you should be able to add version 1.4 of the mod on even Minecraft version 1.28 (though, the interface will likely be broken by this point 😊). I plan to make a version 1.5 soon, so I don't feel the need to update the supported versions on older versions of the mod (also like how I wouldn't mark 1.4 as working on 1.19.5 if I've already released 1.5, since you should just be using the new version).

TL;DR: If the mod is actually broken on some newer version, then obviously I would need to fix it with a new version, but otherwise just assume it works on the latest version of Minecraft and just let me know if you have issues.

Thanks for reading all of that.

**Comment by Madis0** [comment]
Right. Well, similar to what the original commenter said, this is less about whether the mod still works on it and more about whether users and modpack devs can actually find it in search.

But if you're releasing 1.5 soon anyway, I suppose that is fine too.

**Comment by hamarb123** [comment]
I see, so if you made a modpack for "1.19.4" then would it stop you from adding it?

**Comment by Madis0** [comment]
That is correct.

**Comment by hamarb123** [comment]
Ok, I have added it.
